frontend/src/metabase/static-viz/components/ComboChart/ComboChart.tsx

```diff
@@ -0,0 +1,47 @@
+import { init } from "echarts";
+import type { IsomorphicChartProps } from "metabase/static-viz/containers/StaticChart/types";
+import { getCartesianChartModel } from "metabase/visualizations/echarts/cartesian/model";
+import { buildCartesianChartOption } from "metabase/visualizations/echarts/cartesian/option";
+import { sanitizeSvgForBatik } from "metabase/static-viz/lib/svg";
+import { computeStaticComboChartSettings } from "./settings";
+
+const WIDTH = 540;
+const HEIGHT = 360;
+
+export const ComboChart = ({
+  rawSeries,
+  renderingContext,
+}: IsomorphicChartProps) => {
+  const chart = init(null, null, {
+    renderer: "svg",
+    ssr: true,
+    width: WIDTH,
+    height: HEIGHT,
+  });
+
+  const computedVisualizationSettings = computeStaticComboChartSettings(
+    rawSeries,
+    renderingContext,
+  );
+
+  const chartModel = getCartesianChartModel(
+    rawSeries,
+    computedVisualizationSettings,
+  );
+
+  const option = buildCartesianChartOption(
```

Comment:
Nit: I think we can name this `getCartesianChartOption` for consistency with the model function.

For `computeStaticComboChartSettings`, I think it makes sense to use the verb "compute" to distinguish between stored and computed viz settings, but in general I think "get" is a good default.